### PR TITLE
Fixed React and react native contents 

### PR DESCRIPTION
**Pull request by manvith12**
## What does this PR do? 
Add resource(s) | Remove resource(s) 
resolves issue #11570 

### How do we know it's really free?
 it is

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1134,6 +1134,7 @@
 * [React State Management Intermediate JavaScript Course](https://www.youtube.com/watch?v=-bEzt5ISACA) - Jack Herrington (freeCodeCamp)
 * [React State Management using Context API (useContext + useReducer Hooks = Magic)](https://www.youtube.com/watch?v=zxP4oGejqpU) - The Full Stack Junkie
 * [React Testing Library Crash Course](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gm4_-5UsNmLqMosM-dzuvQ) - The Net Ninja
+* [React Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3) - codevolution
```
Comment (Ajay-singh1) [inline courses/free-courses-en.md:1137]:
```suggestion
* [ReactJS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3) - codevolution
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-### Index
+manvith12/free-programming-books### Index
```
Comment (Ajay-singh1) [inline courses/free-courses-en.md:1]:
```suggestion
### Index
```

Comment (Ajay-singh1) [reply courses/free-courses-en.md:1]: Delete this

**Review by Ajay-singh1 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (Ajay-singh1) [reply courses/free-courses-en.md:1137]: Fix the title and fix the linter error.

**Comment by manvith12** [comment]
lint satisfied successfully uff

**Comment by eshellman** [comment]
why not move it to the React section?

**Comment by manvith12** [comment]
> why not move it to the React section?
there was a existing section named react native , should i remove the section and merge both react and react native?

**Comment by eshellman** [comment]
> > why not move it to the React section?
> > there was a existing section named react native , should i remove the section and merge both react and react native?

My understanding was that you think a React resource doesn't belong in the React Native section. Fine, so move it to the React section.

**Comment by manvith12** [comment]
> > > why not move it to the React section?
> > > there was a existing section named react native , should i remove the section and merge both react and react native?
> 
> My understanding was that you think a React resource doesn't belong in the React Native section. Fine, so move it to the React section.
completed

**Comment by eshellman** [comment]
it's not added

**Pull request closed without merging**